pkg/process/procutil/process_linux.go

```diff
@@ -517,19 +517,22 @@ func (p *probe) parseStatusKV(key, value string, sInfo *statusInfo) {
 			sInfo.ctxSwitches.Involuntary = v
 		}
 	case "VmRSS":
-		value := strings.Trim(value, " kB") // trim spaces and suffix "kB"
+		value := strings.TrimSuffix(value, "kB") // trim spaces and suffix "kB"
```

Comment:
To understand this a bit better, why not `strings.TrimSuffix(value, " kB")` to include the extra space.

Reply:
The issue is that the previous code was also making use of the space to trim leading space

Reply:
Understood, but seems like we can get away with performing just one string operation instead of two here.

Reply:
Noticed you closed the PR, still a valuable change though, if we combine into `TrimSuffix` that is.

Reply:
Oh I misunderstood what you meant, the issue I was talking about is that `strings.Trim` trims the suffix but also the prefix, meaning that `strings.TrimSuffix` will never be enough since we also need to trim the prefix spaces

Reply:
So we basically need to strings operations

Reply:
Looking at where `parseStatusKV` is called, it seems that we're already trimming everything on the left of the value:
https://github.com/DataDog/datadog-agent/blob/1ef93ca944af7c16baa210eb5d90d0bb20314937/pkg/process/procutil/process_linux.go#L463-L465

So `value := strings.TrimSuffix(value, " kB")` should work on this case.

I agree that there's no reason to use `Trim` however.

Reply:
Oh, ok. That makes sense then. Mind just updating the comment that I pointed out above then since it's not always true? Otherwise the PR looks good!